src/sentry/tagstore/snuba/backend.py

```diff
@@ -54,6 +59,20 @@ def get_project_list(project_id):
     return project_id if isinstance(project_id, Iterable) else [project_id]
 
 
+def cache_suffix_timeformat(time, key_hash, duration=300):
+    """ Adds jitter based on the key_hash around start/end times for caching snuba queries
+
+        Given a time and a key_hash this should result in a string that remains the same for a duration
+        The end of the duration will be different per key_hash which avoids spikes in the number of queries
+        Must be based on the key_hash so they cache keys are consistent per query
+    """
+    return u"{}:{}:{}".format(
+        time.date().isoformat(),
```

Comment:
Isn't this going to create a different cache key every second, which makes the cache useless ?

Reply:
Nah, this is just the date eg. "2020-01-10" and its so that the key is different per day (I'll add a comment)

Reply:
right. But then, it would mean that at midnight everybody refreshes the cache together.